src/main/java/com/mongodb/jdbc/MongoConnectionProperties.java

```diff
@@ -0,0 +1,46 @@
+package com.mongodb.jdbc;
+
+import com.mongodb.ConnectionString;
+import java.io.File;
+import java.util.logging.Level;
+
+public class MongoConnectionProperties {
+    public ConnectionString connectionString;
+    public String database;
+    public Level logLevel;
+    public File logDir;
+    public String[] clientInfo;
```

Comment:
It was to match the other POJOs such as `JsonSchema` which has public variables.  I have changed them to `protected` and added setters.

Reply:
You don't need getters and setters if the fields are protected because you used them in the same package.
However, I think making them private and only allow setting the value via the constructor makes sense because all the info should be known at the time of creation and not modified.